src/orders/components/OrderCustomerAddressesEditDialog/OrderCustomerAddressesEditDialog.tsx

```diff
@@ -239,11 +176,22 @@ const OrderCustomerAddressesEditDialog: React.FC<OrderCustomerAddressesEditDialo
     customerAddresses
   };
 
+  const getVariant = (action: AddressEditDialogVariant) => {
+    switch (action) {
+      case AddressEditDialogVariant.CHANGE_SHIPPING_ADDRESS:
+        return "shipping";
+      case AddressEditDialogVariant.CHANGE_BILLING_ADDRESS:
+        return "billing";
+    }
+  };
+
```

Comment:
What about change customer, change customer address variants? I think it would be good idea to define explicit union type as return type for this function (and also in utils).

Right now when I change customer, this falls back to billing address as this is how `getAddressEditProps` handles variants other than shipping. E. g. when I change customer I see a checkbox "Set the same for billing address" which suggests that dialog is in shipping mode. However when i disable this checkbox and change address, billing changes together with shipping. Shipping is first address from array, I think. It has to be investigated further.